recipes/pytest-cpp/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "pytest-cpp" %}
+{% set version = "0.4" %}
+{% set sha256 = "3b0d0fe02b3bda961014b23b28b9188be05b01d9dbaf8069292dad1f6b4332e5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - pytest
```

Comment:
FWIW other pytest-plugins have the same dependency during build time:

- [pytest-faulthandler](https://github.com/conda-forge/pytest-faulthandler-feedstock/blob/master/recipe/meta.yaml#L22)
- [pytest-mock](https://github.com/conda-forge/pytest-mock-feedstock/blob/master/recipe/meta.yaml#L22)
- [pytest-qt](https://github.com/conda-forge/pytest-qt-feedstock/blob/master/recipe/meta.yaml#L22)
- [pytest-xdist](https://github.com/conda-forge/pytest-xdist-feedstock/blob/master/recipe/meta.yaml#L23)
- [pytest-benchmark](https://github.com/conda-forge/pytest-benchmark-feedstock/blob/master/recipe/meta.yaml#L20)

But on other hand it doesn't really seem necessary:

- [pytest-cov](https://github.com/conda-forge/pytest-cov-feedstock/blob/master/recipe/meta.yaml#L17)